### PR TITLE
surface buried metadata guidelines

### PR DESCRIPTION
**Pull request by steveoh**
this seems like a good jumping point if we wanted to improve what we had already started. Feel free to hack and slash

### CODE REVIEW

**Comment by gregbunce** [comment]
good find. yes, Erik and I will be working on this in the coming weeks.  thanks for the branch (metadata) to get going!

**Comment by stdavis** [comment]
@gregbunce Did anything ever happen with this? I'm interested in metadata standards to help define the requirements for a [metadata sweeper](https://github.com/agrc/sweeper/issues/33).

**Comment by gregbunce** [comment]
i had no idea this page existed... hence the term "buried".  this is a great start!  and i like the page rename and new location.  

i'd be happy to rework the page and incorporate our notes from [this doc](https://docs.google.com/document/d/1VkXRwfSn6MraI1VeLfei5tg6je4bd2pp_Vh1JuST9xs/edit?usp=sharing) into this page and then make that doc disappear. 

how should i proceed @steveoh ?  i'd like to approve all your initial suggestions and then work from there.  should i approve your changes or make suggestions to your changes?  for me, i'd rather get your changes committed and work fresh from there.

**Comment by steveoh** [comment]
I would suggest checking out this branch and continuing on with it. All of my changes are committed, there are no open suggestions. Do you want to chat offline to get going?

**Comment by gregbunce** [comment]
okay, i see what you're saying.  that makes sense.  for github tracking, should i approve your changes (that you requested my review on) or just let that sit?

**Comment by steveoh** [comment]
Nope, I'd say ignore my pr review request for now.

**Comment by gregbunce** [comment]
okay, i made some changes and i'm happy enough with the metadata page.  it's ready for a review whenever you get a few mins.

b/c i made a bunch of commits (that wasn't my plan, but oh, well), it might be easier to compare the rendered before and after pages.  however, i think they are entirely different now.
before: https://gis.utah.gov/gis-data-sharing/metadata/
after: https://deploy-preview-1274--gis-utah-gov.netlify.app/about/policy/metadata/ 

also, i tried to make the metadata page multipurpose.  
1. i thought we could point sgid contributors to the page when asking them to update their metadata
2. it could be a reference for us agrc folks as far as what we expect and how it should be organized
3. i thought we can put it out there like a "blog post" so others can learn from our method (and expand on it if necessary).  it will make an interesting tweet.

File: about/policy/metadata/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+#### Use Limitations
+{: .text-left}
+
+This section covers limitation and licensing on the data.
+
+The language presented here is for reference only and is populated automatically via our [Sweeper](https://github.com/agrc/sweeper) project.
+
+- *The data herein, including but not limited to geographic data, tabular data, analytical data, electronic data structures or files, are provided "as is" without warranty of any kind, either expressed or implied, or statutory, including, but not limited to, the implied warranties or merchantability and fitness for a particular purpose. The entire risk as to the quality and performance of the data is assumed by the user.*
+
+- *Creative Commons License: This work is licensed under a Creative Commons Attribution 4.0 International License.*
```
Comment (stdavis) [inline about/policy/metadata/index.md:113]: **suggestion**: Format to better reflect the content as it shows up in AGOL.
```suggestion
>The data herein, including but not limited to geographic data, tabular data, analytical data, electronic data structures or files, are provided "as is" without warranty of any kind, either expressed or implied, or statutory, including, but not limited to, the implied warranties or merchantability and fitness for a particular purpose. The entire risk as to the quality and performance of the data is assumed by the user.  
>[![Creative Commons License](https://i.creativecommons.org/l/by/4.0/88x31.png)](http://creativecommons.org/licenses/by/4.0/)  
This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
```

File: about/policy/metadata/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- How reliable and accurate is the dataset?
+  - ex: survey-grade, mapping-grade, etc.
+- What is the general update schedule for this dataset?
+  - ex: monthly, yearly, as needed, etc.
+- Are there any warnings or limitations on the data?
+  - ex: don’t use parcels as a legal boundary depiction
+  - ex: don’t use parcels as a complete dataset for geocoding
+  - ex: don’t use zip codes for demographic generalization
+- Are there any common misconceptions or assumptions about this data that should be further explained?
+
```
Comment (steveoh) [inline about/policy/metadata/index.md:44]: **suggestion**: all the other questions have examples. The milepost one seems like a relevant one to continue the style
```suggestion
  - ex: highway milepost markers are not an accurate length measurement. If a highway is lengthened or shortened, the markers are not adjusted. 
```

File: about/policy/metadata/index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+##### Establish Contact Info
+{: .text-left}
+
+- Who created this dataset?
+- What is the best point of contact for this dataset?
+  - if someone had further questions or would like to get involved, who should they contact?
+
+Your answers to these questions can be plugged into the basic metadata elements (discussed below). It's that simple - answer a few questions, transfer your responses, and voilà, you officially have informative, standardized metadata.
+
+*Note: If you contribute data to Utah's State Geographic Information Database (SGID), we encourage you to use our [metadata form](https://docs.google.com/forms/d/1u7gwdmRN-83Kh5zizi-kHRObeoNyaakM3scPkLT3zKY/edit). It's a great way to update your metadata and make it more discoverable. The form covers all the questions above -and best of all- after you submit your responses, your portion of the process is done! AGRC, in collaboration with the DTS editor, will handle the remaining details.*
```
Comment (steveoh) [inline about/policy/metadata/index.md:54]: **chore**: the convention for italics on our website is a `_`. Consult the readme for other conventions.  
```suggestion
_Note: If you contribute data to Utah's State Geographic Information Database (SGID), we encourage you to use our [metadata form](https://docs.google.com/forms/d/1u7gwdmRN-83Kh5zizi-kHRObeoNyaakM3scPkLT3zKY/edit). It's a great way to update your metadata and make it more discoverable. The form covers all the questions above -and best of all- after you submit your responses, your portion of the process is done! AGRC, in collaboration with the DTS editor, will handle the remaining details._
```

File: about/policy/metadata/index.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+Tags are included in data searches, so at minimum, each SGID dataset should include the stewarding agency (ie: AGRC), SGID, and the appropriate ISO category tag. We also recommend adding any additional, relevant tags that might help with data discoverability. However, there is no need to include words that are contained in the layer's title (these will be omitted).
+
+All words in tags should be title-cased except for articles (“of”, “the”, “and”, etc.) or abbreviations. Each tag should be separated by a comma. Please refer to [Using tags effectively](https://www.esri.com/arcgis-blog/products/arcgis-online/data-management/using-tags-effectively/) for more ideas and information.
+
+`example: AGRC, SGID, Recreation, Paths, Biking, Hiking, Horseback Riding`
+
+#### Summary
+{: .text-left}
+
+The summary should summarize the description in approximately 2 to 3 sentences.
```
Comment (steveoh) [inline about/policy/metadata/index.md:82]: **chore**: remove trailing white space
```suggestion
The summary should summarize the description in approximately 2 to 3 sentences.
```

File: about/policy/metadata/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (steveoh) [reply about/policy/metadata/index.md:54]: good catch, that is the editing link, you posted the resulting form. Please update this!

[the remark this answers is not in the document]
Comment (gregbunce) [reply about/policy/metadata/index.md:54]: you're right.  i'll update that.  thx!

Comment (gregbunce) [reply about/policy/metadata/index.md:54]: oops, busted!  thx

**Comment by gregbunce** [comment]
thanks, ye all for the feedback and suggestions.  the changes should be committed and ready for merge.

**Comment by steveoh** [comment]
🎉

**Pull request merged**